juriscraper/opinions/united_states/state/ill.py

```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8 -*-
 """
 Contact: webmaster@illinoiscourts.gov, 217-558-4490, 312-793-3250
- History:
-   2013-08-16: Created by Krist Jin
-   2014-12-02: Updated by Mike Lissner to remove the summaries code.
-   2016-02-26: Updated by arderyp: simplified thanks to new id attribute identifying decisions table
-   2016-03-27: Updated by arderyp: fixed to handled non-standard formatting
-   2021-11-02: Status validation based on Amendment to Rule 23 rulings
-               https://chicagocouncil.org/illinois-supreme-court-amendment-to-rule-23/
+History:
+  2013-08-16: Created by Krist Jin
+  2014-12-02: Updated by Mike Lissner to remove the summaries code.
+  2016-02-26: Updated by arderyp: simplified thanks to new id attribute identifying decisions table
+  2016-03-27: Updated by arderyp: fixed to handled non-standard formatting
+  2021-11-02: Updated by satsuki-chan: Updated to new page design and updated status validation based on Amendment to Rule 23 rulings
+              https://chicagocouncil.org/illinois-supreme-court-amendment-to-rule-23/
```

Comment:
I would just drop everything after 'and'.

Reply:
Done